src/esockd_listener_sup.erl

```diff
@@ -30,21 +30,19 @@
 -export([ get_options/1
         , get_acceptors/1
         , get_max_connections/1
-        , get_max_conn_rate/3
```

Comment:
We need keep the `get_max_conn_rate/3` and `set_max_conn_rate/3` export. We should try to keep the esockd API as stable as possible.

Reply:
reopen it